projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -25,18 +25,14 @@ import {TUI_SANITIZER} from '@taiga-ui/core';
     },
 })
 export class TuiEditorSocketComponent {
+    private html = '';
+
     @Input()
     set content(content: string) {
```

Comment:
Let's make it declarative, remove setter from `content`, add getter for `html` with `@HostBinding('innerHTML')` and make a private method with `@tuiPure` to sanitize content.

Reply:
unfortunately `@HostBinding('innerHTML')`
doesn't cause a repaint view, if the change detection
wave wasn't triggered

Reply:
But how can `@Input` change without change detection?

Reply:
very strange, I have warning after sanitizing when use `@HostBinding('innerHTML')` 
`WARNING: sanitizing HTML stripped some content, see http://g.co/ng/security#xss`